test/functional/Java8andUp/src/org/openj9/test/reflect/GetMethodTests.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018, 2018 Felipe Pontes
+ * Copyright (c) 2018, 2020 Felipe Pontes
```

Comment:
Just realized this doesn't have an IBM copyright. Please add the following as the next line, with matching formatting.
`Copyright (c) 2020, 2020 IBM Corp. and others`